data/base-spells.yaml

```diff
@@ -110,6 +110,10 @@ cast_messages:
 - You swipe your fingertips
 - ^Your spirit lashes out
 - You open your arms in a hieratic gesture
+- A white static  
```

Comment:
What do you think about `A (white|silver) static` and `A (glowing amber|crimson) mist`?